kbs/src/admin/simple.rs

```diff
@@ -0,0 +1,81 @@
+// Copyright (c) 2025 by NVIDIA.
+// Licensed under the Apache License, Version 2.0, see LICENSE for details.
+// SPDX-License-Identifier: Apache-2.0
+
+use actix_web::{http::header::Header, HttpRequest};
+use actix_web_httpauth::headers::authorization::{Authorization, Bearer};
+use jwt_simple::{
+    claims::NoCustomClaims,
+    common::VerificationOptions,
+    prelude::{Ed25519PublicKey, EdDSAPublicKeyLike},
+};
+use serde::Deserialize;
+use std::path::PathBuf;
+
+use crate::admin::error::*;
+use crate::admin::AdminBackend;
+
+#[derive(Clone, Debug, Default, Deserialize, PartialEq)]
+#[serde(default)]
+pub struct SimpleAdminConfig {
+    pub personas: Vec<SimplePersonaConfig>,
+}
+
+#[derive(Clone, Debug, Default, Deserialize, PartialEq)]
+pub struct SimplePersonaConfig {
+    pub id: String,
```

Comment:
What does this id do? I originally thought there will be an id in the request to tell KBS which public key should be used, but there is not.